sequence_processing_pipeline/GenPrepFileJob.py

```diff
@@ -29,13 +30,18 @@ def __init__(self, run_dir, convert_job_path, qc_job_path, output_path,
                  join(self.output_path, self.run_id, 'Reports'))
 
         for project in project_list:
-            src1 = join(qc_job_path, project, 'filtered_sequences')
```

Comment:
can you use nicer/better names? scr1-2a/b it's not clear.

Reply:
No problem! One moment